docs/network-flow-visibility.md

```diff
@@ -354,6 +354,16 @@ the Flow Aggregator adds the following fields to the flow records.
 | reverseOctetTotalCountFromDestinationNode | 56506         | 133      | unsigned64  |
 | reversePacketDeltaCountFromDestinationNode| 56506         | 134      | unsigned64  |
 | reverseOctetDeltaCountFromDestinationNode | 56506         | 135      | unsigned64  |
+| sourcePodLabels                           | 56506         | 143      | string      |
+| destinationPodLabels                      | 56506         | 144      | string      |
+| throughput                                | 56506         | 145      | unsigned64  |
```

Comment:
what's the unit for these throughput IEs? It may be good to have an additional column in this table with a quick description of the field. We could also remove the Enterprise ID column since it is always the same (we could add a sentence at the beginning of that section to indicate what the Enterprise ID for Antrea is).

Reply:
It's byte/s. I'll add another column "Units" along with "Description". Do you think we need to add description for
1.  all the Antrea IEs(L168-189 + L341-366)? or
2. add it for IEs added by flow-aggregator(L341-366), or
3. only add it for throughput related fields, and leave it blank with other fields

Reply:
> in my experience, throughout in networking is typically measured in bits per second, not bytes per second, but I don't have a strong opinion about whether we should make this change now

Thanks. At the UI dashboard we transform it to bits/s. Yes, I also had this discussion with Yiou previously. We didn't have a strong opinion at that time. This is not a big change. Let me make it now.